firmware/src/hal/sdl/audio_sdl.cpp

```
@@ -0,0 +1,99 @@
+    carried_ -= count;
+    frames -= count;
+  }
+}
+
+}  // namespace
+
+namespace hal {
+
+void start_audio(AudioCallback callback) {
+  audio_callback_ = callback;
+  if (device_ != 0) return;  // already open: the new callback is all that changes
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)